starter/js/atm.js

```diff
@@ -1,43 +1,100 @@
 //Begin with the document ready function
-
+$(document).ready(function(){
+  var amountChecking;
```

Comment:
It's nice that you're defining which variables you'll need at the outset of the program - however, doing so means that the variables you're defining are global (not quite global, but almost). It's generally better practice to define variables in the smallest scope possible - only define them within the functions that you need them. So the "amountChecking" variable should only exist inside the listener that deposits money into the Checking account, etc.